src/views/honeysql/util.clj

```diff
@@ -45,6 +45,11 @@
     (map? wc)  [wc]
     :else      []))
 
+(defn select-tables
```

Comment:
The `select-tables` and the `where-tables` function are very similar and could be generalized into a single higher-order function.